extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/NamespacedExtractionModule.java

```diff
@@ -62,29 +47,21 @@
  */
 public class NamespacedExtractionModule implements DruidModule
 {
-  private static final Logger log = new Logger(NamespacedExtractionModule.class);
   private static final String TYPE_PREFIX = "druid.query.extraction.namespace.cache.type";
-  private static final String STATIC_CONFIG_PREFIX = "druid.query.extraction.namespace";
-  private final ConcurrentMap<String, Function<String, String>> fnCache = new ConcurrentHashMap<>();
-  private final ConcurrentMap<String, Function<String, List<String>>> reverseFnCache= new ConcurrentHashMap<>();
 
   @Override
   public List<? extends Module> getJacksonModules()
   {
     return ImmutableList.<Module>of(
         new SimpleModule("DruidNamespacedExtractionModule")
-        {
-          @Override
-          public void setupModule(SetupContext context)
-          {
-            context.registerSubtypes(NamespacedExtractor.class);
-            context.registerSubtypes(ExtractionNamespace.class);
-          }
-        }
+            .registerSubtypes(
+                ExtractionNamespace.class,
```

Comment:
i don't think it works that way, you want to register subtypes if you are implementing a class from other module like the `LookupExtractorFactory`, otherwise why we don't register all the other classes ?